test/integration/user_account_test.rb

```diff
@@ -190,4 +190,20 @@ class UserAccountTest < ActionDispatch::IntegrationTest
     assert_select 'div.alert', "Last name can't be blank"
     assert_select 'div.alert', "Tos agreement Please accept the Terms and Conditions"
   end
+
+  test "email is sent when a user goes through Forgot Password flow" do
+    post '/users/password',
+    params: { user:
+      {
+        email: 'test@test123.com'
```

Comment:
Let's refer to a fixture here. I know I have not done this in the past but I think moving ahead it's less brittle in case the fixture changes, e.g., "#{users[:user_four].email}"

Reply:
That's a good idea! I pushed the changes just now